src/Core/SharedKernels/DataCollection/DataCollection/Events/Interview/GroupPropagated.cs

```diff
@@ -1,20 +0,0 @@
-﻿using System;
-using WB.Core.SharedKernels.DataCollection.Events.Interview.Base;
-
-namespace WB.Core.SharedKernels.DataCollection.Events.Interview
-{
-    [Obsolete("Since v6.0")]
-    public class GroupPropagated : InterviewPassiveEvent

```

Comment:
do we ignore all events in db?

Reply:
We agreed to do it. 
There is a workaround - delete questionnaire and reimport.